acceptance/.beaker.yml

```diff
@@ -0,0 +1,15 @@
+---
+ssh:
+  keys:
+    - id_rsa_acceptance
+    - ~/.ssh/id_rsa-acceptance
+xml:                         true
+timesync:                    false
+repo_proxy:                  true
+add_el_extras:               false
+forge_host:                  forge-aio01-petest.puppetlabs.com
+'master-start-curl-retries': 30
+log_level:                   debug
+preserve_hosts:              onfail
+helper:                      ./lib/helper.rb
+options_file:                ./config/aio/options.rb
```

Comment:
Why do you add the options file to specifically be `config/aio/options`? Will this be overwritten by `--options-file` when we run beaker?

Reply:
I added aio so that `bx beaker exec pre-suite` just works, where `pre-suite` is a generic name for "all of the tests in the currently configured presuite"

Reply:
Hmm, this does makes sense, but I'm worried it might trip folks up later, because it's not necessarily expected behavior? Especially when we get git testing working. However, I don't want to block on this, so I'm definitely fine leaving it as is